examples/chart/teleport/README.md

```diff
@@ -1,29 +1,43 @@
 
 ## Introduction
 
-This chart deploys Teleport components to your cluster via a Kubernetes `Deployment`.
+This chart deploys Teleport Community or Enterprise components to your cluster via a Kubernetes `Deployment`.
 
 By default this chart is configured as follows:
 
-- 1 replica
+- Enterprise Edition of Teleport
+- 1 instance (replica) of Teleport
+- Directory Storage with no persistent storage.  
 - Record ssh/k8s exec and attach session to the `emptyDir` of the Teleport pod
-- The assumed externally accessible hostname of Teleport is `teleport.example.com`
-- Teleport is accessible only from within your cluster. You need `kubectl port-forward` for external access.
-- TLS is enabled by default on the Proxy
+- The assumed externally accessible hostname of Teleport is `teleport.example.com` 
+- Teleport is accessible only from within your cluster. You need `kubectl port-forward` for external access. Change the Service type in `values.yaml` to options such as LoadBalancer to make it externally accessible.
+- TLS is enabled by default on the Proxy 
 
-See the comments in the default `values.yaml` and also the Teleport documentation for more options.
+
+The `values.yaml` is configurable for multiple options including:
+- Using the Community edition of Teleport (Set license.enabled to false)
+- Using self-signed TLS certificates (Set proxy.tls.usetlssecret to false)
+- Using a specific version of Teleport (See image.tag)
+- Using persistent or high availability storage (See below example).  Persistent or high availability storage is recommended for production usage. 
+- Increasing the replica count for multiple instances (Using High Availability configuration)
+
+See the comments in the default `values.yaml` and also the [Teleport documentation](https://gravitational.com/teleport/docs/) for more options.   
 
 ## Prerequisites
 
 - Kubernetes 1.10+
-- A Teleport license file stored as a Kubernetes Secret object(See below)
+- Teleport license for Enterprise deployments
+- TLS Certificates or optionally use self-signed certificates
+
+### Prepare a Teleport Enterprise license file
+
 
-### Prepare the license file
+If you are deploying the Enterprise version you will require the license file as a secret available to Teleport. To use the community edition of Teleport simply set `enabled: false` under the `license:` settings in `values.yaml`.
 
-Download the `license.pem` from the Teleport dashboard, and then rename it to the filename that this chart expects:
+Download the `license.pem` from the Teleport dashboard, and then <b>rename it to the filename</b> that this chart expects:
```

Comment:
We should add a link here Teleport dashboard -> https://dashboard.gravitational.com/web/login